### PR TITLE
#13 Add contributing guidelines

### LINKED ISSUE #13: Add contributing guidelines
It can be done based on https://github.com/blog/1184-contributing-guidelines

### PR DESCRIPTION
**Pull request by mkordas**
This work closes #13.

### CODE REVIEW

**Comment by nejdaw** [comment]
@mkordas please connect this PR with an Issue, so Waffle.io recognises it as a contribution rather than separate story.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Commit Naming Convention
+
```
Comment (nejdaw) [inline CONTRIBUTING.md:2]: We could add here information about linking PR with Issues.

Comment (mkordas) [inline CONTRIBUTING.md:2]: Done, please review

**Comment by mkordas** [comment]
@dawidnejman - done

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+All pull requests should be linked to issues with additional comment in PR description. This enables Waffle to move
+them as a single unit to "In Progress" state and assign pull request creator to the referenced issue.
+
+There are two ways of linking:
+
+1. Using GitHub's closing keywords (fixes/closes/resolves) prior to a GitHub issue number in pull request description
+automatically closes the issue after merge. Example: `This work closes #13.` in PR description.
+2. Using Waffle's connect keywords (connect to/connects to/connected to/connects) prior to a GitHub issue number in pull
+request description will not close the issue after merge. Example: `This work connects to #13.` in PR description.
```
Comment (balawama) [inline CONTRIBUTING.md]: No dot is needed :)

Comment (mkordas) [inline CONTRIBUTING.md]: fixed

**Pull request merged**